### ISSUE TITLE
貌似卡住了

### ISSUE THREAD

**Issue by Conliq** [thuml/Time-Series-Library#242]
运行scripts/long_term_forecast/ILI_script/TimesNet.sh的第一个示例，分完数据集的长度后等待十几min也没有新的print

Args in experiment:
Namespace(activation='gelu', anomaly_ratio=0.25, batch_size=32, c_out=7, checkpoints='./checkpoints/', d_ff=768, d_layers=1, d_model=768, data='custom', data_path='national_illness.csv', dec_in=7, des='Exp', devices='0,1,2,3', distil=True, dropout=
0.1, e_layers=2, embed='timeF', enc_in=7, factor=3, features='M', freq='h', gpu=0, inverse=False, is_training=1, itr=1, label_len=18, learning_rate=0.0001, loss='MSE', lradj='type1', mask_rate=0.25, model='TimesNet', model_id='ili_36_24', moving_av
g=25, n_heads=8, num_kernels=6, num_workers=10, output_attention=False, p_hidden_dims=[128, 128], p_hidden_layers=2, patience=3, pred_len=24, root_path='./dataset/illness/', seasonal_patterns='Monthly', seq_len=36, target='OT', task_name='long_term
_forecast', top_k=5, train_epochs=10, use_amp=False, use_gpu=True, use_multi_gpu=False)
Use GPU: cuda:0
>>>>>>>start training : long_term_forecast_ili_36_24_TimesNet_custom_ftM_sl36_ll18_pl24_dm768_nh8_el2_dl1_df768_fc3_ebtimeF_dtTrue_Exp_0>>>>>>>>>>>>>>>>>>>>>>>>>>
train 617
val 74   
test 170

**Comment by wuhaixu2016** [comment]
您好，如果使用linux的话，我没有遇到过这个问题。如果使用windows的话，需要将num_workers设置为0.
